nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -80,6 +88,12 @@
 
     public static final String SERVICE_WITHOUT_IMPLEMENTATION_MESSAGE = "This service implementation does not implement deprecated API.";
 
+    /** @since 11.1 **/
+    public static final String COMMENT_RELATED_TEXT_ID = "commentRelatedTextId_%s";
+
+    /** @since 11.1 **/
+    public static final String ANNOTATION_RELATED_TEXT_ID = "annotationRelatedTextId_%s";
```

Comment:
Could you merge the two fields? As discussed, it brings more complexity for an unknown feature need.